probe-dictionary/index.html

```diff
@@ -51,6 +51,9 @@
           <li>
             <a class="nav-link" href="https://telemetry.mozilla.org/"><i class="fa fa-home"></i> Telemetry portal</a>
           </li>
+          <li>
+            <span class="permalink-control"></span>
```

Comment:
We can include the HTML inline here, I think. It'll save us some JS execution time on load, as well as making it so it doesn't "pop in" after the rest of the page has painted.

Reply:
@chutten , Yeah ! That sounds right. I will try to implement it in this way now. Thanks!